site/about/glossary.qmd

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Glossary"
+title: "ValidMind Glossary"
```

Comment:
Same here, we use sentence style capitalization (as per our spiffy new style guide).

(Also, we have more than just ValidMind glossary terms, so we might want to revisit this title. E.g. none of the MRM terms are specific to us, they are industry-standard terms.)

```suggestion
title: "ValidMind glossary"
```